aries-site/src/pages/components/tabs.mdx

```diff
@@ -33,6 +36,14 @@ the browser location.** This will allow proper active styling and routing to fun
 
 **If using Tabs as a navigation structure, don't use them in conjunction with other navigation structures.** Combining multiple navigation structures creates a confusing experience for the user. It can lead to difficulties in navigating through an application and in ability to remember where desired content lives.
```

Comment:
I found wanting a little bit more context for this guidance. Is there an example we can provide as a "don't"? Or provide more context on what other "navigation structures" should not be paired with Tabs? Does that mean Tabs with other nested Tabs?

Reply:
This content is existing content and I have not touched it. This can be addressed in a different audit of Tab content if desired.

Reply:
That's fair.